packages/types/src/chip/index.ts

```diff
@@ -90,7 +90,7 @@ export const UserTapSchema = z.object({
   ownerPsiPublicKeyLink: z.string().nullable(),
   ownerUserData: JsonSchema.nullable(),
   timestamp: z.coerce.date(),
-  chipPublicKeySignature: ChipPublicKeySignatureSchema.nullable(),
+  chipPublicKeySignature: z.string().nullable(),
```

Comment:
Why was this schema removed?

Reply:
weird thing where currently in the DB it's stringified and this was slightly easier than other fixes